analyses/cell-type-wilms-tumor-06/notebook/04_annotation_Across_Samples_exploration.Rmd

```diff
@@ -0,0 +1,310 @@
+---
+title: "Annotation exploration for `r params$scpca_project_id`"
+author: "Maud PLASCHKA"
+date: '2024-08-13'
+params:
+  scpca_project_id: "SCPCP000006"
+  sample_id: "SCPCS000176"
+  seed: 12345
+output: 
+  html_document: 
+    toc: yes
+    toc_float: yes
+    code_folding: hide
+    highlight: pygments
+    df_print: paged
+---
+
+```{r setup, include=FALSE}
+
+knitr::opts_chunk$set(echo = TRUE,
+                      message=FALSE,
+                      warnings=FALSE
+                      )
+
+```
+
+
+## Introduction
+
+
+The aim is to explore the annotations and label transfers for all of the samples in `r params$scpca_project_id`.
```

Comment:
For this module, it will always be the same project :) 
```suggestion
The aim is to explore the annotations and label transfers for all of the samples in SCPCP000006.
```